### ISSUE TITLE
Feture Request - Rename Layer

### ISSUE THREAD

**Issue by meford777** [DWilliames/google-sheets-sync-figma#243]
I would like to be able to rename a layer based upon a cell in the spreadsheet. 

For example, I'm making static online ads where the location changes. I have the text layer pulling in the data, but i would like the component layer to be renamed to whatever's in the "Location" column of the spreadsheet as well. This is so that when i go to export all of the frames they're named based on the location.

Thanks, your plugin is great!

**Comment by donatasben** [comment]
Yes, I would like this functionality too! Very useful to be able to rename layers automatically! Could use some placeholder syntax as usual (like "My custom {name} for this instance")
